src/video_codec.cpp

```diff
@@ -0,0 +1,12 @@
+#include "video_codec.hpp"
+#include <iostream>
+LiveVideoFeed::videoCodec::videoCodec(const std::string &code) {
+    if (code.length() == 4) {
+        codeID = code;
```

Comment:
Use `codeID = std::copy(code);` To clarify that code is being copied. I think string::operator=(std::string str) copies it by default, but this explicitly states that code is copied.